src/pages/MyStorage/index.tsx

```diff
@@ -58,6 +75,7 @@ export default function Home() {
 
   return (
     <div className="flex flex-col flex-1">
+      <Appbar onSearchChange={(e) => setSearchTerm(e.target.value)} />
```

Comment:
hey @borjasolerme 
remove this one
 
and try filter func on appbar component :)